xapian-core/net/tcpserver.h

```diff
@@ -105,6 +105,11 @@ class XAPIAN_VISIBILITY_DEFAULT TcpServer SOCKET_INITIALIZER_MIXIN {
     /** Accept a single connection, service requests on it, then stop.  */
     void run_once();
 
+    /** Function overload to call fork-based implementation.
+     *  Accept a single connection, service requests on it in a child process,
+     *  then stop.  */
+    void run_once(bool);
```

Comment:
I used this trick to make a minimum change to make it work(so I don't break anything). Please suggest if you have a clean fix in mind.

Reply:
I think it's better to refactor it so the forking is handled outside of `run_once()` (which then means that `run_once()` would then works the same way cross-platform).  I've pushed that change as 4ae447e7382ff4983fe15d02663b28edc01d3479 so you can just drop this commit.